### PR TITLE
Fix typos in runtime expval notebook

### PR DESCRIPTION
**Pull request by glingi**
- Typo
line 84 :  expecation -> expectation
line 127, 448 : devitation -> deviation
line 354 : excute -> execute
line 593 : heavy-ouputs -> heavy-outputs
line 662 : meaasurements -> measurements
lint 688 : expection -> expectation

- Grammer
line 39 : where an user -> where a user
line 66, 534 : the all -> all
line 491 : Lets -> Let's

### Details and comments
Fixes #

### CODE REVIEW

File: docs/tutorials/sample_expval_program/qiskit_runtime_expval_program.ipynb [hunk 1]
```diff
@@ -63,7 +63,7 @@
     "- `circuits`: A single QuantumCircuit or list of QuantumCircuits to be executed on the target backend.\n",
     "\n",
     "\n",
-    "- `expectation_operators`: The operators we want to evaluate.  These can be strings of diagonal Pauli's, eg, `ZIZZ`, or custom operators defined by dictionaries. For example, the projection operator on the all ones state of 4 qubits is `{'1111': 1}`.\n",
```
Comment (kt474) [inline docs/tutorials/sample_expval_program/qiskit_runtime_expval_program.ipynb:66]: I think this is actually correct, "all ones state" is one entity so the "the" is necessary

File: docs/tutorials/sample_expval_program/qiskit_runtime_expval_program.ipynb [hunk 2]
```diff
@@ -531,7 +531,7 @@
    "id": "29268312",
    "metadata": {},
    "source": [
-    "The first two projectors should be nearly $0.50$ as they tell use the probability of being in the all zeros and ones states, respectively, which should be 50/50 for our GHZ state.  The final expectation value of `ZZZZ` should be one since this is a GHZ over an even number of qubits.  It should go close to zero for an odd number."
```
Comment (kt474) [inline docs/tutorials/sample_expval_program/qiskit_runtime_expval_program.ipynb:534]: same as above, also use -> us

Comment (glingi) [reply docs/tutorials/sample_expval_program/qiskit_runtime_expval_program.ipynb:534]: I think so too.

**Review by kt474 — APPROVED** [review]
Thanks @glingi! - In the future please checkout from the main branch when creating PRs

**Comment by glingi** [comment]
> Thanks @glingi! - In the future please checkout from the main branch when creating PRs

Oh.. I'm sorry for that. I will keep that in mind.

**Pull request merged**